assets/js/interactivity/index.js

```diff
@@ -1,13 +1,18 @@
+// @ts-nocheck
+
+/**
+ * Internal dependencies
+ */
 import registerDirectives from './directives';
 import { init } from './router';
 import { rawStore, afterLoads } from './store';
```

Comment:
We import `rawStore` to pass it to `afterLoad` callbacks. This is not required anymore as you could use `store()` to get a reference to any store you need.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-blocks/pull/11071/commits/826bb6e75fe2726fabf1ee2db7c012c8af61509a.

Reply:
I was thinking that `afterLoad` is not needed anymore. Is it?

Reply:
Uhm. We added that API to initialize stuff that depended on the store's _readiness_. Do we have something in the new API that replaces that? 🤔

Reply:
> We added that API to initialize stuff that depended on the store's readiness

Hmm... after all the `store()` calls have been executed, you mean?

If we look at this: https://github.com/woocommerce/woocommerce-blocks/blob/interactivity-api-refactor-product-button-with-new-store-api/assets/js/atomic/blocks/product-elements/button/frontend.tsx#L128-L146

The need for `afterLoad` is not justified anymore. You can just do this and it works fine:

```js
const { state } = store("woo", {
  // ....
});

subscribe(() => {
  const cartData = select(storeKey).getCartData();
  const isResolutionFinished =
    select(storeKey).hasFinishedResolution("getCartData");
  if (isResolutionFinished) {
    state.cart = cartData;
  }
}, storeKey);

requestIdleCallback(() => {
  if (!state.hasCartLoaded) {
    select(storeKey).getCartData();
  }
});
```

Reply:
Oh, right. 😄  The `afterLoad` API was intended to get the store reference and use it in the code you wanted to execute.

Now I see why it's not needed anymore. I'll remove anything related to the `afterLoad` callbacks.

Reply:
As you said, the only use case that I see is to have a way to wait until all the stores have been merged, so if you are accessing some part of the store that it's not defined in your file, it's there when you access `state`.

But we don't have any use case for that now, so my vote goes for removing it, and we'll evaluate it once we see the need for it again.